mcs/build/profiles/winaot.make

```diff
@@ -28,6 +28,8 @@ PROFILE_MCS_FLAGS = \
 	-d:DISABLE_REMOTING \
 	-d:DISABLE_COM \
 	-d:WIN_PLATFORM \
+	-d:FEATURE_CONSOLE \
+	-d:NO_MULTIPLE_APPDOMAINS \
```

Comment:
It seems to only be disabled on monotouch_watch and monotouch_tv, but it is enabled on monotouch.

I haven't looked what exactly this define disables, but if monotouch/testing_full_aot  doesn't need it (they seem to compile and run fine) then I'd not set it.